secret.go

```diff
@@ -86,25 +86,7 @@ func (r *SecretService) New(ctx context.Context, body SecretNewParams, opts ...o
 	return
 }
 
-// Lists secrets with optional filtering.
-//
-// Use this method to:
-//
-// - View all project secrets
-// - Filter secrets by project
-//
-// ### Examples
-//
-// - List project secrets:
-//
-//	Shows all secrets for a project.
-//
-//	```yaml
-//	filter:
-//	  projectIds: ["b0e12f6c-4c67-429d-a4a6-d9838b5da047"]
-//	pagination:
-//	  pageSize: 20
-//	```
+// ListSecrets
```

Comment:
Missing docs

Reply:
Seems to be a regression from https://github.com/gitpod-io/gitpod-next/issues/5866

Reply:
Fixing this in https://github.com/gitpod-io/gitpod-next/pull/6507.